src/compute-plots.jl

```diff
@@ -349,29 +368,46 @@ function add_attributes!(::Type{T}, attr, kwargs) where {T}
             end
         else
             add_input!(attr, k, value)
+            attr[k].value = RefValue{Any}(value)
         end
 
         # Hack-fix variable type
         if haskey(abstract_type_init, k)
             attr[k].value = abstract_type_init[k]
         end
     end
+    if !haskey(attr, :model)
+        add_input!(attr, :model, Mat4f(I))
```

Comment:
At this point it should still be a Mat4d. Also maybe not as an input but a Computed that gets updated by transformations

Reply:
Yeah, I should have left a TODO, this definitely needs to change.